scripts/test.ps1

```diff
@@ -59,7 +59,6 @@ $Script:TPT_Configuration = $Configuration
 $Script:TPT_SourceFolders =  @("test")
 $Script:TPT_TargetFrameworks =@($TPT_TargetFrameworkCore, $TPT_TargetFrameworkFullCLR)
 $Script:TPT_TargetRuntime = $TargetRuntime
-$Script:TPT_SkipProjects = @("testhost.UnitTests")
```

Comment:
Please don't delete TPT_SkipProjects functionality. Let's keep it as 911 :) We can remove testhost.UnitTests from the list.